os/various/lib_scsi.h

```diff
@@ -110,15 +110,15 @@ typedef enum {
  * @brief   Represents SCSI sense data structure.
  * @details See SCSI specification.
  */
-typedef struct PACKED_VAR {
+typedef struct {
```

Comment:
As a result of redefinition errors. I don't know if this is the correct way to solve this.